Tests/Microsoft.AppCenter.Test.Windows/Channel/ChannelTest.cs

```diff
@@ -3,41 +3,40 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AppCenter.Channel;
-using Microsoft.AppCenter.Storage;
+using Microsoft.AppCenter.Ingestion;
 using Microsoft.AppCenter.Ingestion.Models;
 using Microsoft.AppCenter.Ingestion.Models.Serialization;
+using Microsoft.AppCenter.Storage;
 using Microsoft.AppCenter.Test.Storage;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 
 namespace Microsoft.AppCenter.Test.Channel
 {
+    using Channel = Microsoft.AppCenter.Channel.Channel;
+
     [TestClass]
     public class ChannelTest
     {
         private AggregateException _unobservedTaskException;
-        private readonly MockIngestion _mockIngestion = new MockIngestion();
-        private Microsoft.AppCenter.Channel.Channel _channel;
+        private Mock<IIngestion> _mockIngestion;
```

Comment:
Since `IIngestion` was changed and `Moq` works fine here, I didn't rewrite manual mock